htlcswitch/link_test.go

```diff
@@ -637,6 +637,160 @@ func testChannelLinkMultiHopPayment(t *testing.T,
 	}
 }
 
+func TestChannelLinkInboundFee(t *testing.T) {
+	t.Parallel()
+
+	t.Run("negative", func(t *testing.T) {
+		t.Parallel()
+
+		bobInboundFee := models.InboundFee{
+			Base: -500,
```

Comment:
Perhaps a test for over-payment of inbound fees and total negative fees could be useful as well.

Reply:
Good one, added